src/User.js

```diff
@@ -51,18 +53,23 @@ class User {
   }
 
   filterAvailableRooms = (desiredRoomType) => {
+    console.log("hey hey", this.availableRooms)
     let wantedRooms = this.availableRooms.filter(availableRoom => {
       if(availableRoom.roomType === desiredRoomType) {
         return availableRoom;
       } 
     })
-    if(wantedRooms.length > 0) {
-      return wantedRooms;
-    } else {
-      return `Sorry for the inconvenience! Please try another room type!`
-    }
+      this.wantedRooms = wantedRooms;
   }
 
+  throwError = () => {
+    let desired = this.wantedRooms;
+    console.log("here", desired)
+    if(desired.length === 0) {
+       this.message = `I am SO SORRY! Please forgive me! I beg your pardon, and I promise if you pick another room, you can have one a tiny home. On me.`
```

Comment:
Check sentence for grammar for error message, but otherwise, a very apologetic message to the customer!